src/LibraryPage.tsx

```diff
@@ -42,7 +38,6 @@ const LibraryPage: React.FC = () => {
   const [user, setUser] = useState<User>();
   const [modalOpen, setModalOpen] = useState(false);
   const [isNewBook, setIsNewBook] = useState(true);
-  const [books, setBooks] = useState<Book[]>([]);
```

Comment:
I think yours and my merges might have messed things up. I deleted one "books" declaration, and this deletes the other haha. We need at least one in there